azure-pipelines.yaml

```diff
@@ -229,7 +229,7 @@ jobs:
       - task: Bash@3
         displayName: 'Send slack message - master branch'
         condition: |
-          and(succeeded(), eq(variables['isMain'], true), eq(variables['isAutoTriggered'], true))
+          and(succeeded(), eq(variables['isMain'], true))
```

Comment:
I can't comment on the line in this but line 224 what if someone re-triggers a build in Azure DevOps? won't that stop notifications working? probably not a blocker

Reply:
Yeah, if someone were to manually trigger it then it wouldn't send the notifications. I suspect it would work if you selected the PR commit in DevOps when running the pipeline but the only options in this pipeline are the branches rather than the PR refs. They can just push an empty commit to their pr (not as simple as clicking run I grant you)

Reply:
let's see how we go, can adjust later, this is much better anyway